tools/roslaunch/src/roslaunch/substitution_args.py

```diff
@@ -46,6 +46,7 @@
 
 import rosgraph.names
 import rospkg
+import loader
```

Comment:
Is this referring to the relative module `roslaunch.loader`? Then please use absolute import names (optionally with `from` and updating the usage) and insert them in alphabetical order. E.g.:

    import rosgraph.names
    from roslaunch.loader import convert_value
    import rospkg